### DISCUSSION TITLE
如何在所有页面中引入一段自定义js代码，用于前端事件控制

### DISCUSSION THREAD

**Discussion by Paper-Dragon** [vuepress-theme-hope/vuepress-theme-hope#3610] [Q&A] [answered]
如何在所有页面中引入一段自定义js代码，用于前端事件控制

比如加入访问某页面弹窗或者按下某个键弹窗

**Comment by Mister-Hope** [comment, marked as answer]
https://theme-hope.vuejs.press/zh/cookbook/customize/effect.html#%E6%B7%BB%E5%8A%A0%E7%89%B9%E6%95%88

**Reply by Paper-Dragon** [reply]
感谢，已经用上了

**Reply by 006hjy** [reply]
> https://theme-hope.vuejs.press/zh/cookbook/customize/effect.html#%E6%B7%BB%E5%8A%A0%E7%89%B9%E6%95%88

页面404了，可以再发一下吗？

**Reply by Mister-Hope** [reply]
https://theme-hope.vuejs.press/guide/customize/effect.html
